operator-pipeline-images/operatorcert/pyxis.py

```diff
@@ -35,6 +35,15 @@ def _get_session() -> requests.Session:
     api_key = os.environ.get("PYXIS_API_KEY")
     cert = os.environ.get("PYXIS_CERT_PATH")
     key = os.environ.get("PYXIS_KEY_PATH")
+    env = os.environ.get("ENVIRONMENT")
+
+    proxies = {}
+    # If it's external preprod
+    if env != "prod" and api_key:
```

Comment:
Can you please provide a documentation link in a code so we know where does this configuration come from?

Reply:
Ok, done!